tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/RevalidationServiceImpl.java

```diff
@@ -228,10 +228,12 @@ public ConnectionInfoDto buildTcsConnectionInfo(Long personId) {
 
     // $1 and $2 will be injected with the parenthesized match sub patterns from the RegEx
     final String whereClause = String.format("where $1.$2 = %s", personId);
+    final String individualPlacementClause = String.format("and pl.traineeId = %s", personId);
 
     final String query = sqlQuerySupplier
         .getQuery(SqlQuerySupplier.TRAINEE_CONNECTION_INFO)
         .replaceAll(WHERE_CLAUSE_MACRO_REGEX, whereClause)
+        .replace("INDIVIDUALPLACEMENTCLAUSE", individualPlacementClause)
```

Comment:
The WHERECLAUSE was modified and made more flexible to accommodate field names & table aliases.
If we are going to stick with a separate template for this clause, I'll ask that this uses that same method to reduce the likelihood of the table alias `pl` and field name changing in one place but not the other.

Reply:
Cool, I could change it to an ANDWHERE clause

Reply:
Have used the nested query as suggested